packages/lib/time.ts

```
@@ -117,12 +117,23 @@
-  for (let key in obj) {
+export const convertDatesInObject = (obj: any): any => {
+  if (obj === null || typeof obj !== "object") {
+    return obj; // Return the value if obj is not an object
+  }
+  if (Array.isArray(obj)) {
+    // Handle arrays separately
+    return obj.map((item) => convertDatesInObject(item));
+  }
+  const newObj: any = {};
+  for (const key in obj) {
     if ((key === "createdAt" || key === "updatedAt") && !isNaN(Date.parse(obj[key]))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)